web/src/components/user/NewUserFields/index.tsx

```diff
@@ -144,8 +136,46 @@ export default function NewUserFields({
         {...register('password', { minLength: 8 })}
       ></FieldPassword>
       } */}
+      {short &&  // not required
+        <Accordion icon={<IoOptions/>} title={t("user.optionalFields")} >
+
+            <FieldText
+            name="email"
+            label={t('user.email')}
+            explain={t('user.emailExplain')}
+            classNameInput="squared"
+            placeholder={t('user.emailPlaceHolder')}
+            validationError={errors.email}
+            {...register('email')}
+          ></FieldText>
+          <FieldText
```

Comment:
are you sure that we want to request the phone on registration if its an invitation?